app/v1/models/request/amendPeriodic/PeriodAllowableExpenses.scala

```diff
@@ -0,0 +1,79 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.amendPeriodic
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class PeriodAllowableExpenses(consolidatedExpenses: Option[BigDecimal],
+                                   costOfGoodsAllowable: Option[BigDecimal],
+                                   paymentsToSubcontractorsAllowable: Option[BigDecimal],
+                                   wagesAndStaffCostsAllowable: Option[BigDecimal],
+                                   carVanTravelExpensesAllowable: Option[BigDecimal],
+                                   premisesRunningCostsAllowable: Option[BigDecimal],
+                                   maintenanceCostsAllowable: Option[BigDecimal],
+                                   adminCostsAllowable: Option[BigDecimal],
+                                   businessEntertainmentCostsAllowable: Option[BigDecimal],
+                                   advertisingCostsAllowable: Option[BigDecimal],
+                                   interestOnBankOtherLoansAllowable: Option[BigDecimal],
+                                   financeChargesAllowable: Option[BigDecimal],
+                                   irrecoverableDebtsAllowable: Option[BigDecimal],
+                                   professionalFeesAllowable: Option[BigDecimal],
+                                   depreciationAllowable: Option[BigDecimal],
+                                   otherExpensesAllowable: Option[BigDecimal])
+
+object PeriodAllowableExpenses {
+
+  implicit val reads: Reads[PeriodAllowableExpenses] = (
+    (JsPath \ "consolidatedExpenses").readNullable[BigDecimal] and
+      (JsPath \ "costOfGoodsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "paymentsToSubcontractorsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "wagesAndStaffCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "carVanTravelExpensesAllowable").readNullable[BigDecimal] and
+      (JsPath \ "premisesRunningCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "maintenanceCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "adminCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "businessEntertainmentCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "advertisingCostsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "interestOnBankOtherLoansAllowable").readNullable[BigDecimal] and
+      (JsPath \ "financeChargesAllowable").readNullable[BigDecimal] and
+      (JsPath \ "irrecoverableDebtsAllowable").readNullable[BigDecimal] and
+      (JsPath \ "professionalFeesAllowable").readNullable[BigDecimal] and
+      (JsPath \ "depreciationAllowable").readNullable[BigDecimal] and
+      (JsPath \ "otherExpensesAllowable").readNullable[BigDecimal]
+    )(PeriodAllowableExpenses.apply _)
```

Comment:
these reads can just be generic json reads since there is no field name transformations